WAL/WAL/Screen/Create/Controller/CreateInformationViewController.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  CreateInformationViewController.swift
+//  WAL
+//
+//  Created by 배은서 on 2022/06/20.
+//
+
+import UIKit
+
+import Then
+import WALKit
+
+class CreateInformationViewController: UIViewController {
+    
+    // MARK: - Properties
+    
+    private let informationView = UIView().then {
+        $0.backgroundColor = .white100
+        $0.makeRound(radius: 20)
+    }
+    
+    private lazy var closeButton = UIButton().then {
+        $0.setImage(WALKit.WALIcon.btnDelete.image, for: .normal)
+        $0.addTarget(self, action: #selector(touchUpCloseButton), for: .touchUpInside)
+    }
+    
+    private let titleLabel = UILabel().then {
+        $0.text = "왈소리 만들기란?"
+        $0.font = WALKit.WALFont.title2.font
```

Comment:
요거 상관은 없는데 그냥 
```suggestion
        $0.font = WALFont.title2.font
```
이렇게 써도 가능합니다 ~